kubernetes/linux/Dockerfile.multiarch

```diff
@@ -1,8 +1,8 @@
 # Default base images. If you update them don't forgot to update variables in our build pipelines. Default values can be found in internal wiki. External can use ubuntu 18.04 and golang 1.18.3
 ARG GOLANG_BASE_IMAGE=

```

Comment:
golang is now at 1.20 might be worth upgrading?

Reply:
> golang is now at 1.20 might be worth upgrading?

> @pfrcks , Do you see any urgency to add as part of this upgrade otherwise we can handle in subsequent release?

Reply:
we can add it seperately as well